yarn.lock

```diff
@@ -25360,7 +25360,7 @@ subscriptions-transport-ws@^0.9.18, subscriptions-transport-ws@^0.9.19:
     symbol-observable "^1.0.4"
     ws "^5.2.0 || ^6.0.0 || ^7.0.0"
 
-sucrase@^3.18.0, sucrase@^3.18.2:
+sucrase@^3.18.0:
```

Comment:
Maybe merge this block with the one below

Reply:
Can't, the jest plugin still pulls in the old version :<

Reply:
Hm maybe I'm blind, they looked semver-compatible